README.adoc

```diff
@@ -127,17 +130,20 @@ The [hotspot=from file=0]`FROM` instruction initializes a new build stage, which
+
+It is recommended to label your Docker images with the [hotspot=label file=0]`LABEL` command, as the information will help you manage your images. See https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#label[Best practices for writing Dockerfiles^] for more information.
 
-During the build, use a single `ADD` command instead of using multiple `COPY` commands to transfer configuration files and the application because the application is packaged into an archive. It eliminates multiple layers as the Docker image is created.
+The [hotspot=copy-config hotspot=copy-war file=0]`COPY` instructions are structured as `COPY` [hotspot=config-userID hotspot=war-userID file=0]`[--chown=<user>:<group>]` [hotspot=inventory-config hotspot=inventory-war file=0]`<source>` [hotspot=config hotspot=config-apps file=0]`<destination>`. 
+They copy local files into the specified destination within your Docker image.
+In this case, the `inventory` server configuration files located at `src/main/liberty/config` are copied to the destination directory [hotspot=config file=0]`/config/`.
+The `inventory` application WAR file [hotspot=inventory-war file=0]`inventory.war`, created from running `mvn package` previously, is copied to the destination directory [hotspot=config-apps file=0]`/config/apps`.
 
-The [hotspot=add file=0]`ADD` instruction is structured as [hotspot=add file=0]`ADD` [hotspot=userID file=0]`[--chown=<user>:<group>]` [hotspot=source file=0]`<source>` [hotspot=destination file=0]`<destination>`. 
-It copies local files and directories into the specified destination within your Docker image. If the source is an archive, the [hotspot=add file=0]`ADD` instruction unpacks it into the destination directory.
-In this case, the `inventory` application archive [hotspot=source file=0]`inventory.tar.gz`, created from running `mvn install` previously, is extracted to the destination directory [hotspot=destination file=0]`/opt/ol`. 
+The [hotspot=copy-config hotspot=copy-war file=0]`COPY` instructions need to use the user ID [hotspot=config-userID hotspot=war-userID file=0]`1001` and group [hotspot=config-userID hotspot=war-userID file=0]`0` because the `open-liberty` image runs by default with the `USER 1001` (non-root) user for security purposes. Otherwise, the files and directories that are copied over are owned by the root user.
 
-The [hotspot=add file=0]`ADD` instruction needs to use the user ID [hotspot=userID file=0]`1001` and group [hotspot=userID file=0]`0` because the `open-liberty` image runs by default with the `USER 1001` (non-root) user for security purposes. Otherwise, the files and directories that are copied over are owned by the root user.
+The [hotspot=configure-sh file=0]`RUN configure.sh` command should be placed at the last to get a pre-warmed Docker image. It will improve the startup time of running your Docker container.
```

Comment:
Change
"The [hotspot=configure-sh file=0]`RUN configure.sh` command should be placed at the last to get a pre-warmed Docker image. It will improve the startup time of running your Docker container."
To
"Place the [hotspot=configure-sh file=0]`RUN configure.sh` command at the end to get a pre-warmed Docker image. It improves the startup time of running your Docker container."